### ISSUE TITLE
Food Handling Room - Pentry vs Pantry

### ISSUE THREAD

**Issue by Chike-O** [Azure/opendigitaltwins-building#37]
I believe the model at opendigitaltwins-building/Ontology/Space/Room/FoodHandlingRoom/Pentry.json is supposed to represent a food pantry; was it named "pentry" intentionally?  Will it be updated to reflect the Pantry?

**Comment by hammar** [comment]
Good catch! That could be a misspelling, or possibly a mistranslation ("Pentry" is a Swedish word for "Kitchenette"; the RealEstateCore models were originally developed in Sweden). I'll look into it and see which is the case (and modify the source OWL models and the resulting generated DTDL models accordingly).
